src/components/SideTray/style.css

```diff
@@ -0,0 +1,56 @@
+.sideTray {
+  width: var(--rvr-side-tray-width);
+  height: 100vh;
+  background-color: var(--rvr-white);
+  position: fixed;
+  top: 0;
+  right: 0;
+  transition: transform 0.3s cubic-bezier(0, 0.52, 0, 1);
+  overflow: auto;
+  z-index: var(--rvr-off-canvas-drawer-z-index);
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.2);
+}
+
+.hide {
+  transform: translate3d(400px, 0, 0);
+}
+
+.show {
+  transform: translate3d(0, 0, 0);
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  overflow: hidden;
+  height: 100%;
+}
+
+.content {
+  flex-grow: 1;
+  overflow: auto;
+  padding: 0;
+}
+
+.backdrop {
+  visibility: visible;
+  border: none;
+  user-select: none;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  z-index: var(--rvr-off-canvas-drawer-z-index) - 1;
+  background-color: rgba(0, 0, 0, 0.3);
+  overflow: auto;
+  opacity: 10;
+  transition: opacity 150ms var(--rvr-easeInOutQuad);
```

Comment:
The variables for timing functions are a good start. Wonder if we could come up with a generic pattern to inform when to use what? (I just posted about [this](https://uxdesign.cc/the-ultimate-guide-to-proper-use-of-animation-in-ux-10bd98614fa9) in eng-guild-front-end)

I was thinking something like

Timing function variables
* timing-fade
* timing-move-in
* timing-move-out

Durations
* slow
* medium
* fast

For this element, since it's pretty big, we can do "slow timing-fade" for the backdrop and "slow timing-move-in"/"slow timing-move-out" for the actual tray. We'd want to go back and use a "fast" duration for buttons, and we could even come back later and adjust speeds based on media queries.

Reply:
I'm not gonna do this at the moment; i'm not even convinced 150ms is a quantity worth transitioning over